docparse/output_walkthrough.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Output Walkthrough'
+description: 'Walking through a document using Aryn DocParse'
+icon: 'person-walking'
+---
+
+## Introduction
+
+In [this example](https://colab.research.google.com/drive/120STOQ5wGIpFUbgS7Gcld_PVW-CfLWOg#scrollTo=bTu2a1K-F19w), we'll use DocParse to extract data from an NTSB report. We'll go through the important code snippets below to see what's going on. For the purposes of this tutorial, we focus on page 2 of the document, given below.
+
+<Frame>
+  <img src="../images/59_page2.png" />
+</Frame>
+
+## Getting Parsed Document
+
+### Making a Call to DocParse
+
+```python text_demo.py
+with open(file_name, 'rb') as file:
+  ## Make a call to the partitioning service and set extract_images to true.
+  partitioned_file = partition_file(file, aryn_api_key, extract_images=True, extract_table_structure=True, use_ocr=False, selected_pages=[2])
+```
+We use the `partition_file` function from `aryn_sdk.partition` to extract text and images from the document. The `aryn_api_key` is your API key from Aryn. Set `extract_images` and `extract_table_structure` to `True` to extract images and tables. Set `use_ocr` to `False` to avoid OCR and use embedded text. We use `selected_pages=[2]` to focus on page 2.
+
+### Viewing the JSON Output
+
+```json output.json
+{'status': ['Incremental status will be shown here during execution.',
```

Comment:
is there a use case where someone would want to use 'status' programatically? Not saying we should get into it in this walkthrough page but curious.

Reply:
It's unlikely that it could be needed, and there's a good reason for it to be removed entirely. The point of this page is to just show and go through the whole output.